src/frontend/screens/Library/components/GameCard/index.css

```diff
@@ -106,6 +120,12 @@
   padding: var(--space-xs-fixed);
 }
 
+.gameCard.justPlayed > .icons {
+  justify-content: space-between;
+  grid-template-columns: 0.15fr 0.15fr 0.15fr;
```

Comment:
```suggestion
  grid-template-columns: 1fr 1fr 1fr;
```

Reply:
![Screenshot from 2023-06-06 15-27-03](https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/assets/74495920/6d40714d-ed08-444d-bb57-6ca6e9438234)

The suggested change will end up looking like this though. Do you prefer this?

Reply:
Or we could discuss the ideal layout in Discord with the others' feedback.